website/templates/nav.mako

```diff
@@ -82,21 +82,8 @@
             %else :
             <li class="dropdown sign-in" data-bind="with: $root.signIn">
                 <div class="col-sm-12">
-                    <a href="${web_url_for('auth_login')}?sign_up=True" class="btn btn-success btn-top-signup m-r-xs">Sign up</a>
-                    <button type="button" class="btn btn-info btn-top-login p-sm dropdown-toggle" data-toggle="dropdown" aria-expanded="false">
-                        Sign in <span class="caret hidden-xs"></span>
-                    </button>
-                    <ul class="dropdown-menu" id="menuLogin" role="menu">
-                        <form class="form" id="signInForm" data-bind="submit: submit" action="${login_url}" method="POST">
-                            <div class="form-group"><input id="email" class="form-control" type="email" data-bind="value: username" name="username" placeholder="Email" aria-label="Username"></div>
-                            <div class="form-group"><input name="password" id="password" class="form-control" type="password" placeholder="Password" data-bind="value: password" aria-label="Password"></div>
-                            <div class="form-group"><button type="submit" id="btnLogin" class="btn btn-block btn-primary">Login</button></div>
-                            %if enable_institutions:
-                                <div class="text-center m-b-sm"> <a href="/login/?campaign=institution">Login through your institution  <i class="fa fa-arrow-right"></i></a></div>
-                            %endif
-                            <div class="text-center m-b-sm"> <a href="/forgotpassword/">Forgot password?</a></div>
-                        </form>
-                    </ul>
+                    <a href="${web_url_for('auth_register')}" class="btn btn-success btn-top-signup m-r-xs">Sign Up</a>
+                    <a href="${login_url}" class="btn btn-info btn-top-login p-sm">Sign In</a>
```

Comment:
Note for CR: I had planned to replace `${login_url}` with `${web_url_for('auth_login')` which serves as a entry point view. However, this breaks several redirections back from CAS. These redirections are calculated in `routes.py` as globals.